onedal/tests/utils/_dataframes_support.py

```diff
@@ -96,6 +97,11 @@ def get_df_and_q(dataframe: str):
                 df_and_q.append(pytest.param(dataframe, queue.values[0], id=id))
         return df_and_q
 
+    if "np_sycl" in dataframe_filter_:
+        # sycl queue param is not None.
+        # Designed for interfaces that utilize NumPy inputs with a DPCTL queue,
+        # enabling offloading to specific SYCL devices.
+        dataframes_and_queues.extend(get_df_and_q("numpy_and_queue"))
```

Comment:
@icfaust
1)  what do you think about the naming and this approach specifically? Does it make sense to leave only `numpy_and_queue` instead `np_sycl`?

2) Maybe it is better to use param: `numpy_with_queue=True`

I mean:
```
get_dataframes_and_queues("numpy", numpy_and_queue=True))
```

instead of:
```
get_dataframes_and_queues("numpy,np_sycl"))
```

Reply:
the second option could be generalized for all dataframes that doesn't support sycl_queue, but when sycl_queue is provided